### DISCUSSION TITLE
Why particles system is moved to cuda only?

### DISCUSSION THREAD

**Discussion by trsh** [NVIDIA-Omniverse/PhysX#84] [Q&A] [answered]
The prior library FleX worked also on dx11/dx12. Now its vendor locked to cuda cores making the technology irrelevant for video games. Why?

**Comment by amoravanszky** [comment, marked as answer]
I could count the number of games that have used GPU simulation with Flex on one hand.  The typical user of GPU PhysX is doing robotics, AI or industrial simulation work, and in these fields the CUDA platform is well established.
